hare/hare.go

```diff
@@ -229,37 +229,47 @@ func (h *Hare) collectOutput(ctx context.Context, output TerminationOutput) erro
 
 	var pids []types.ProposalID
 	if output.Completed() {
+		consensusCount.WithLabelValues(success).Inc()
 		h.WithContext(ctx).With().Info("hare terminated with success", layerID, log.Int("num_proposals", output.Set().Size()))
 		set := output.Set()
+		postNumProposals.Add(float64(set.len()))
 		pids = make([]types.ProposalID, 0, set.len())
 		for _, v := range set.elements() {
 			pids = append(pids, v)
 		}
 	} else {
+		consensusCount.WithLabelValues(failure).Inc()
 		h.WithContext(ctx).With().Info("hare terminated with failure", layerID)
 	}
 
 	hareOutput := types.EmptyBlockID
 	if len(pids) > 0 {
 		// fetch proposals from peers if not locally available
 		if err := h.fetcher.GetProposals(ctx, pids); err != nil {
+			blockGenCount.WithLabelValues(failFetch).Inc()
```

Comment:
i think it is better to initialize this metrics with a label and then use it here, e.g.

```
var failFetchCount = blockGenCount.WithLabelValues...
failFetchCount.Inc()
```

Reply:
thanks. done for the metrics in this PR